rest-api-spec/src/main/resources/rest-api-spec/test/search/340_keyword_doc_values.yml

```diff
@@ -0,0 +1,46 @@
+---
+"search on keyword fields with doc_values enabled":
+  - do:
+      indices.create:
+        index: test
+        body:
+          mappings:
+            properties:
+              "some_keyword":
+                type: "keyword"
+                index: true
+                doc_values: true
```

Comment:
The more valuable check is that it works with `index:false`, but you covered that with unit tests.